bin/check-rspec.rb

```diff
@@ -77,18 +77,24 @@ class CheckRspec < Sensu::Plugin::Check::CLI
          long: '--handler HANDLER',
          default: 'default'
 
+  option :source,
```

Comment:
from what I see this is just a client name, if so this should probably be called `:proxy_client` also I would remove the short option, and make the long match the option name. in rspec `-p` is for profile adding to extra confusion. Let's be clear about our intent with our variable and option names so they make more sense.